### ISSUE TITLE
Feature Request: Start Menu

### ISSUE THREAD

**Issue by MoDerPo** [overviewer/Minecraft-Overviewer#1858]
I suggest a simple batch file with the following cmd: `start cmd /k title Minecraft Overviewer` along with an icon. That way one could make add it to the start menu and open it just like an app.
[icon+bat.zip](https://github.com/overviewer/Minecraft-Overviewer/files/5352125/icon%2Bbat.zip)

**Comment by LeadMagnet** [comment]
Are you referring to the "overviewer" command itself? Or a shortcut to the output "index.html" file?

The first is not useful at all, as it's a command line tool. The normal use method is just fine.

The second would be interesting, but again doing this for all use cases might make some people irritated. I my case, the server that renders the map is not the same server that hosts the output HTML. Since (normally) the output folder never changes no matter how many times you run it, or which version is used, manually creating a shortcut is a "one and done" action.

Not sure it's worth the dev time.

**Comment by MoDerPo** [comment]
Yes I mean the command line. So you could just go to the Windows search and type in "Overviewer" to open it up and start typing your command
